### PR TITLE
initial version

### PR DESCRIPTION
**Pull request by dtrawins**

### CODE REVIEW

File: bundle_k8s/manifests/openvino-operator.clusterserviceversion.yaml [hunk 1]
```diff
@@ -522,5 +522,5 @@ spec:
   maturity: alpha
   provider:
     name: Intel
-  version: 1.1.0
-  replaces: openvino-operator.v1.0.0
+  version: 1.2.0
+  replaces: openvino-operator.v1.1.0
```
Comment (pgladkows) [inline bundle_k8s/manifests/openvino-operator.clusterserviceversion.yaml:526]: could this name be consistent with the version?

File: bundle/manifests/openvino-operator.clusterserviceversion.yaml [hunk 2]
```diff
@@ -609,5 +609,5 @@ spec:
   maturity: alpha
   provider:
     name: Intel
-  version: 1.1.0
-  replaces: openvino-operator.v1.0.0
+  version: 1.2.0
+  replaces: openvino-operator.v1.1.0
```
Comment (pgladkows) [inline bundle/manifests/openvino-operator.clusterserviceversion.yaml:613]: could this name be consistent with the version?

File: bundle_k8s/manifests/openvino-operator.clusterserviceversion.yaml (hunk 1 above)
Comment (dtrawins) [reply bundle_k8s/manifests/openvino-operator.clusterserviceversion.yaml:526]: it is consistent. new version will be called 1.2.0 to replace 1.1.0.

**Review by pgladkows — APPROVED** [review]

File: bundle/manifests/intel.com_notebooks.yaml [hunk 3]
```diff
@@ -15,7 +15,7 @@ spec:
     - name: v1alpha1
       schema:
         openAPIV3Schema:
-          description: Notebook is the Schema representing the Image streams for RHODS JupyterHub notebooks
+          description: Notebook is the Schema representing the Image streams for AI JupyterHub notebooks
```
Comment (mzegla) [inline bundle/manifests/intel.com_notebooks.yaml:18]: Isn't the full name "OpenShift AI" ?

File: bundle/manifests/openvino-operator.clusterserviceversion.yaml [hunk 4]
```diff
@@ -146,7 +146,7 @@ spec:
               - 'urn:alm:descriptor:com.tectonic.ui:fieldDependency:models_settings.single_model_mode:true'
           - description: >-
               Target device to be used to execute the inference or the plugin name for handling multiple devices.
-              It could be format like `CPU`, `GPU`, `HDDL`, `AUTO`, `MULTI:CPU,GPU`.
+              It could be format like `CPU`, `GPU`, `AUTO`, `MULTI:CPU,GPU`.
```
Comment (mzegla) [inline bundle/manifests/openvino-operator.clusterserviceversion.yaml:149]: What about HETERO?

**Review by mzegla — APPROVED** [review]

**Pull request merged**